pkg/kubelet/cm/container_manager_linux.go

```diff
@@ -190,6 +199,38 @@ const (
 	KernelTunableModify KernelTunableBehavior = "modify"
 )
 
+// InitQOS creates the top level qos cgroup containers
+// We currently create containers for the Burstable and Best Effort containers
+// All guaranteed pods are nested under the RootContainer by default
+// InitQOS is called only once during kubelet bootstrapping.
+func InitQOS(rootContainer string, subsystems *cgroupSubsystems) (QOSContainersInfo, error) {
+	cm := NewCgroupManager(subsystems)
+	// Top level for Qos containers are created only for Burstable
+	// and Best Effort classes
+	qosClasses := [2]qos.QOSClass{qos.Burstable, qos.BestEffort}
+
+	// Create containers for both qos classes
+	for _, qosClass := range qosClasses {
+		//get the containers absolute name
+		absoluteContainerName := path.Join(rootContainer, string(qosClass))
```

Comment:
Yes I thought it would be a good idea to have that check over here: https://github.com/kubernetes/kubernetes/blob/master/cmd/kubelet/app/server.go#L360

WDYT??